src/ui-actions/move-conv-msg-modal/move-conv-msg.jsx

```diff
@@ -26,7 +26,6 @@ const MoveConvMessage = ({ selectedIDs, isMessageView, isRestore, deselectAll, o
 	const dispatch = useDispatch();
 	const createSnackbar = useContext(SnackbarManagerContext);
 	const [currentFolder, setCurrentFolder] = useState('');
-	const [currentFolderName, setCurrentFolderName] = useState('');
 	const [folderDestination, setFolderDestination] = useState(currentFolder || {});
```

Comment:
can you please check if one of the state from currentFolder or folderDestination can be removed ? ,because I think both are same and I guess we don't need the currentFolder because we are moving so destination is required